src/riscv-legacy-integration.adoc

```diff
@@ -220,11 +220,8 @@ When an extended CSR is used with another CSR instruction (<<CSRRWI>>,
 * The final address is updated as specified in <<extended_CSR_writing>> for an
 XLEN write.
 * XLEN bits are read from the capability address field and written to an output
-*x* register.
-
-There is _no distinction_ between Legacy and Capability Mode in this case -
-XLEN access is always performed, and the assembly syntax always uses *x*
-registers.
+*x* register if the mode is Capability. In Legacy mode, CLEN bits are read from
```

Comment:
elsewhere I capitalise legacy Mode. it would be nicer to say

"... in Capability Mode" "...in Legacy Mode"

Reply:
Fixed